src/CartFee.php

```diff
@@ -23,12 +24,14 @@ class CartFee
      *
      * @param $amount
      * @param $taxable
+     * @param $tax
      * @param array $options
      */
-    public function __construct($amount, $taxable, $options = [])
+    public function __construct($amount, $taxable = false, $tax = 0, $options = [])
```

Comment:
If you don't put default value, you wouldn't be able to use the shortened version.